### ISSUE TITLE
Feature Request - Add ability to edit flight name and description

### ISSUE THREAD

**Issue by deksprime** [PX4/flight_review#136]
Changing the flight name and description might be a very useful addition to the flight review functionality. After the log has been uploaded the logs are visually inspected by the uploader and the resulting insights are currently not being annotated within the log page. I think this might be useful for both the uploader and the general database of the flight review.

**Comment by bkueng** [comment]
I agree this would be very useful. However the best way to do that and to avoid arbitrary edits is to have user logins, which is a bigger topic.

**Comment by deksprime** [comment]
Oh... That's correct, anyone shouldn't be able to edit so I guess this feature is waiting on the user login function implementation.
